pinot-common/src/main/java/org/apache/pinot/common/utils/config/QueryOptionsUtils.java

```diff
@@ -182,6 +185,19 @@ public static Integer getNumReplicaGroupsToQuery(Map<String, String> queryOption
     return checkedParseIntPositive(QueryOptionKey.NUM_REPLICA_GROUPS_TO_QUERY, numReplicaGroupsToQuery);
   }
 
+  public static List<Integer> getOrderedPreferredReplicas(Map<String, String> queryOptions) {
+    String orderedPreferredReplicas = queryOptions.get(QueryOptionKey.ORDERED_PREFERRED_REPLICAS);
+    if (orderedPreferredReplicas == null) {
+      return Collections.emptyList();
+    }
+    String[] replicas = orderedPreferredReplicas.split("\\|");
```

Comment:
need to fix a proper way to fix the Calcite parsing error. Previously test a single element only.